UnitTests/BraintreeTestShared/MockAPIClient.swift

```diff
@@ -13,6 +13,7 @@ public class MockAPIClient: BTAPIClient {
     public var postedAnalyticsEvents : [String] = []
     public var postedPayPalContextID: String? = nil
     public var postedLinkType: String? = nil
+    public var postedShouldSendAPIRequestLatency: Bool = false
```

Comment:
We can remove this

Reply:
Done: https://github.com/braintree/braintree_ios/pull/1292/commits/58768861720cbaa7e3c1b16aba7fb6102b82b0c7